lib/driver.js

```diff
@@ -45,14 +100,46 @@ export default class RokuDriver extends BaseDriver {
     return [sessionId, caps];
   }
 
-  async deleteSession () {
+  async deleteSession() {
     log.info('Ending Roku session');
     // always press home to end
     await this.roku_pressKey({key: 'Home'});
     await super.deleteSession();
   }
+
+  performActions = performActions;
+  click = click;
+  focus = focus;
+  focusElement = focusElement;
+  makeNextFocusMove = makeNextFocusMove;
+  setValue = setValue;
+  performActionWithKeyboard = performActionWithKeyboard;
+  setValueByEcp = setValueByEcp;
+  setValueByKeyboard = setValueByKeyboard;
+  removeApp = removeApp.bind(this);
+  activateApp = activateApp.bind(this);
+  roku_activateApp = roku_activateApp;
+  roku_appUI = roku_appUI;
+  roku_deviceInfo = roku_deviceInfo;
+  roku_getApps = roku_getApps;
+  roku_installApp = roku_installApp;
+  roku_pressKey = roku_pressKey;
+  installApp = installApp.bind(this);
```

Comment:
some of these functions must be bound and I haven't been able to understand why.